Nos/Views/RepliesView.swift

```diff
@@ -84,7 +84,7 @@ struct RepliesView: View {
     var body: some View {
         VStack {
             ScrollView(.vertical) {
-                LazyVStack {
+                VStack {
```

Comment:
How many replies we can show before making it a problem for VStack? Hopefully is something ridiculously large

Reply:
I'm not sure, but I haven't had problems yet. The docs for LazyVStack are pretty emphatic to not use it until you see VStack having performance problems so that's what I'm doing.

Reply:
Fair enough